### PR TITLE
モデルを作成

### PR DESCRIPTION
**Pull request by highwide**
## 目的

wikiで定義したとおりモデルを作ります
## todo
- [x] モデル作成

いったんこんな感じかと！

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20150621061920) do
+
+  create_table "daily_progresses", force: :cascade do |t|
```
Comment (highwide) [inline db/schema.rb:16]: **[rails_best_practices]**
- always add db index (daily_progresses => [practice_id])

File: app/models/daily_progress.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class DailyProgress < ActiveRecord::Base
```
Comment (highwide) [inline app/models/daily_progress.rb]: __[reek]__
* [IrresponsibleModule] has no descriptive comment

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    t.integer  "practice_id"
+    t.date     "done_at"
+    t.integer  "goal"
+    t.integer  "progress"
+    t.datetime "created_at",  null: false
+    t.datetime "updated_at",  null: false
+  end
+
+  create_table "practices", force: :cascade do |t|
```
Comment (highwide) [inline db/schema.rb:25]: **[rails_best_practices]**
- always add db index (practices => [user_id])

File: app/models/practice.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+class Practice < ActiveRecord::Base
```
Comment (highwide) [inline app/models/practice.rb]: __[reek]__
* [IrresponsibleModule] has no descriptive comment

File: app/models/user.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+class User < ActiveRecord::Base
```
Comment (highwide) [inline app/models/user.rb]: __[reek]__
* [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150621061709_create_users.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration
```
Comment (highwide) [inline db/migrate/20150621061709_create_users.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150621061709_create_users.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  def change
```
Comment (highwide) [inline db/migrate/20150621061709_create_users.rb:2]: **[reek]**
- [TooManyStatements] has approx 6 statements
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150621061709_create_users.rb [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+    create_table :users do |t|
```
Comment (highwide) [inline db/migrate/20150621061709_create_users.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: db/migrate/20150621061845_create_practices.rb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+class CreatePractices < ActiveRecord::Migration
```
Comment (highwide) [inline db/migrate/20150621061845_create_practices.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150621061845_create_practices.rb [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :practices do |t|
```
Comment (highwide) [inline db/migrate/20150621061845_create_practices.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: db/migrate/20150621061845_create_practices.rb [hunk 11]
```diff
@@ -0,0 +1,11 @@
+class CreatePractices < ActiveRecord::Migration
+  def change
```
Comment (highwide) [inline db/migrate/20150621061845_create_practices.rb:2]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150621061920_create_daily_progresses.rb [hunk 12]
```diff
@@ -0,0 +1,12 @@
+class CreateDailyProgresses < ActiveRecord::Migration
```
Comment (highwide) [inline db/migrate/20150621061920_create_daily_progresses.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150621061920_create_daily_progresses.rb [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+  def change
```
Comment (highwide) [inline db/migrate/20150621061920_create_daily_progresses.rb:2]: **[reek]**
- [TooManyStatements] has approx 6 statements
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150621061920_create_daily_progresses.rb [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+    create_table :daily_progresses do |t|
```
Comment (highwide) [inline db/migrate/20150621061920_create_daily_progresses.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: app/models/user.rb [hunk 15]
```diff
@@ -0,0 +1,7 @@
...
+  has_many  :practices
```
Comment (highwide) [inline app/models/user.rb]: __[rails_best_practices]__
* remove trailing whitespace

Comment (highwide) [inline app/models/user.rb:2]: **[rails_best_practices]**
- remove trailing whitespace

File: app/models/daily_progress.rb [hunk 16]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (highwide) [inline app/models/daily_progress.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: app/models/practice.rb [hunk 17]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (highwide) [inline app/models/practice.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: app/models/user.rb [hunk 18]
```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
```
Comment (highwide) [inline app/models/user.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

**Comment by ta1kt0me** [comment]
LGTM !!!!

**Pull request merged**